beacon-chain/core/helpers/committee.go

```diff
@@ -29,23 +34,22 @@ var committeeCache = cache.NewCommitteesCache()
 //        MAX_COMMITTEES_PER_SLOT,
 //        len(get_active_validator_indices(state, epoch)) // SLOTS_PER_EPOCH // TARGET_COMMITTEE_SIZE,
 //    ))
-func CommitteeCountAtSlot(state *pb.BeaconState, slot uint64) (uint64, error) {
-	epoch := SlotToEpoch(slot)
-	count, err := ActiveValidatorCount(state, epoch)
-	if err != nil {
-		return 0, errors.Wrap(err, "could not get active count")
-	}
-	var committeePerSlot = count / params.BeaconConfig().SlotsPerEpoch / params.BeaconConfig().TargetCommitteeSize
+func CommitteeCountAtSlot(validatorCount uint64) uint64 {
```

Comment:
Maybe change this to `activeValidatorCount`?

Reply:
Yeah.. what is this?